src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+      description: GluuServer configuration data document Endpoint.It contains non-standard OpenID Connect discovery metadata supported by the Gluu server.
+      operationId: getConfiguration      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: GluuConfiguration
+                $ref: '#/components/schemas/GluuConfiguration'   
+                #description:Client GluuAttribute by Dn(Distinguished Name) based on Authorization Scope.                                         
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        403:
+          description: Access denied.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'  
+  /introspection:
+    get:
+      tags:
+      - developers
+      summary: The Introspection OAuth 2 Endpoint responds to HTTP GET requests from token holders. 
+      description: The Introspection Endpoint takes a single parameter representing the token and returns a JSON document representing the token meta information.
+      operationId: introspectGet
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          description: Client Authorization details that contains the access token along with other details.
+          schema:
+            type : string
+        - name: token
+          in: query
+          required: true
+          #description: ID of pet that needs to be updated         
+          schema:
+            type : string
+        - name: token_type_hint
+          in: query
+          required: true
+          #description:ID Token previously issued by the Authorization Server being passed as a hint about the End-User.
+          schema:
+            type : string            
+        - name: response_as_jwt
+          in: query
+          required: true
+          #description: Value true indicates that response to be jwt
+          schema:
+            type : string        
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: IntrospectionResponse
+                $ref: '#/components/schemas/IntrospectionResponse'                  
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'   
+  /introspection(post):
```

Comment:
We don't have `/introspection(post)`, please stick to endpoints definition.

Reply:
Rectified